monad-loops.cabal

```diff
@@ -36,6 +36,15 @@ Library
   exposed-modules:      Control.Monad.Loops
   if flag(base4)
     cpp-options:        -Dbase4
-    build-depends:      base >= 4 && <5
+    build-depends:      base >= 4 && < 5, tasty, tasty-hunit
   else
-    build-depends:      base >= 2 && < 4
+    build-depends:      base >= 2 && < 4, tasty, tasty-hunit
```

Comment:
This is here for convenience (to run the unit tests through `cabal repl` instead of through `cabal test`).